code/__DEFINES/time.dm

```diff
@@ -4,6 +4,9 @@
 ///displays the current time into the round, with a lot of extra code just there for ensuring it looks okay after an entire day passes
 #define ROUND_TIME(...) ( "[world.time - SSticker.round_start_time > MIDNIGHT_ROLLOVER ? "[round((world.time - SSticker.round_start_time)/MIDNIGHT_ROLLOVER)]:[worldtime2text()]" : worldtime2text()]" )
 
+///Returns the time that has passed since the game started
+#define STATION_TIME_PASSED (world.time - SSticker.round_start_time)
```

Comment:
If it's not constant, it shouldn't look like one

```suggestion
#define STATION_TIME_PASSED(...) (world.time - SSticker.round_start_time)
```

This requires that you use `STATION_TIME_PASSED()`.